main/res/values/strings.xml

```diff
@@ -1564,6 +1565,12 @@
     <string name="search_bookmark_list">Bookmark list</string>
     <string name="search_pocket_is_new">new</string>
     <string name="search_pocket_and_bookmark_loading">Loading a list of Pocket Queries and bookmark lists</string>
+    <string name="search_bookmark_loading">Loading bookmark lists</string>
+    <string name="search_bookmark_select">Select bookmark lists</string>
+    <string name="search_bookmark_new">Create new bookmark lists</string>
```

Comment:
Should probably be singular "Create new bookmark list"

Reply:
Thanks, good catch... Fixed.